geoprocessing/src/utils/file.utils.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * File Utils
+ */
+
+import { MulterOptions } from '@nestjs/platform-express/multer/interfaces/multer-options.interface';
+import { createReadStream, readdirSync } from 'fs';
+
+/**
+ * ES5 import to avoid TS complaining
+ */
+const multer = require('multer');
+
+/**
+ * Options for Multer
+ */
```

Comment:
JSDoc isn't any helpful there. Consider removing it or providing description on its purpose.